modules/node/src/util/logger.ts

```diff
@@ -4,21 +4,21 @@ export class CLogger extends Logger {
   public cxt: string;
   constructor(context: string) {
     super();
-    this.cxt = context || "";
+    this.cxt = context || "?";
```

Comment:
why was this change made?

Reply:
At some point I was seeing the `can't access property cxt of undefined` error and wanted a visual indication of a missing context but I don't think this change actually helped anything..

Reply:
I still see this sometimes too.

Reply:
Actually I just got the following logs so this `"?"` might be a clue 🔎
```
[Nest] 180   - 12/04/2019, 9:33 AM   [AuthService] Got channel 0x2D9077260075B11eA0874c629AEb3B45b2531DdA: undefined
[Nest] 180   - 12/04/2019, 9:33 AM   [AuthService] Acquiring a lock for a multisig w/out a channel: lock.acquire.0x2D9077260075B11eA0874c629AEb3B45b2531DdA +1ms
?
[Nest] 180   - 12/04/2019, 9:33 AM   [LockService] Using lock ttl of 90 seconds
```